### PR TITLE
Issue#31

### LINKED ISSUE #31: Home Page: Add more details about athena
The home page is a little empty. We should add some bullet points about what athena is and what it does.

### PR DESCRIPTION
**Pull request by PeterETschaekofske**
Details Added to Athena Home Page

Fixes #31 for real this time

### CODE REVIEW

File: src/Athena/Views/Home/Home.cshtml [hunk 1]
```diff
@@ -5,14 +5,34 @@
...
-            Welcome to the Athena Scheduling Assistant!
-        </h1>
-        <p>
-            This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
-        </p>
+<div class="notLoggedIn" style="padding-top:5em">
+    <h1>
+        Welcome to the Athena Scheduling Assistant!
+    </h1>
+    <p>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Let's make this more readable on larger screens by adding `class="flow-text"`

File: src/Athena/Views/Home/Home.cshtml [hunk 2]
```diff
@@ -5,14 +5,34 @@
...
+        This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
+    </p>
+    <div class="row center">
+        <div class="col s4">
+            <i class="large material-icons">person_add</i>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Lets color the icons by adding these classes: `blue-text text-accent-3`

File: src/Athena/Views/Home/Home.cshtml [hunk 3]
```diff
@@ -5,14 +5,34 @@
...
+
+            <h4>Security</h4>
+            <p>Uses your Google or Microsoft account for easy authentication. We provide a means of keeping your information safe by using the same authentication services as most universities.</p>
+        </div>
+        <div class="col s4">
+            <i class="large material-icons">event_available</i>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Lets color the icons by adding these classes: `blue-text text-accent-3`

File: src/Athena/Views/Home/Home.cshtml [hunk 4]
```diff
@@ -5,14 +5,34 @@
...
+
+            <h4>Scheduling & Things</h4>
+            <p> Schedule the classes you need and view your week at a glance. Manage the classes you've taken and programs you're enrolled in. You can even see the progress of your program completion.</p>
+        </div>
+        <div class="col s4">
+            <i class="large material-icons">mood</i>
```
Comment (nlowe) [inline src/Athena/Views/Home/Home.cshtml]: Lets color the icons by adding these classes: `blue-text text-accent-3`

File: src/Athena/Views/Home/Home.cshtml (hunk 1 above)
Comment (nlowe) [reply src/Athena/Views/Home/Home.cshtml]: Lets also give this `<p>` a `margin-bottom` of `5rem`

Comment (PeterETschaekofske) [reply src/Athena/Views/Home/Home.cshtml]: thats fixed.

File: src/Athena/Views/Home/Home.cshtml (hunk 2 above)
Comment (PeterETschaekofske) [reply src/Athena/Views/Home/Home.cshtml]: thats fixed

File: src/Athena/Views/Home/Home.cshtml (hunk 3 above)
Comment (PeterETschaekofske) [reply src/Athena/Views/Home/Home.cshtml]: thats fixed

File: src/Athena/Views/Home/Home.cshtml (hunk 4 above)
Comment (PeterETschaekofske) [reply src/Athena/Views/Home/Home.cshtml]: thats fixed

**Pull request merged**